packages/protocol/scripts/create-stable-token.ts

```diff
@@ -0,0 +1,265 @@
+import * as fs from 'fs'
+
+const KEY_TO_BE_REPLACED_IN_TEMPLATES = '{TICKER}'
+
+const StableTokenTemplate = `pragma solidity ^0.5.13;
+
+import "./StableToken.sol";
+
+contract StableToken{TICKER} is StableToken {
```

Comment:
Do you think this would be cleaner if each of these templates were instead a function returning a string using builtin string interpolation (`` `${ticker}` ``)?

Reply:
+1

Reply:
Done.